modules/codecs/test/resources/golden.json

```diff
@@ -0,0 +1,1459 @@
+{
```

Comment:
This is used in a golden test, in order to prevent accidental breakage of protocol